docs/storybook/stories/agents-ui/AudioVisualizerWave.stories.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import { StoryObj } from '@storybook/react-vite';
+import {
+  AgentSessionProvider,
+  useMicrophone,
+} from '../../.storybook/lk-decorators/AgentSessionProvider';
+import { AudioVisualizerWave, AudioVisualizerWaveProps } from '@agents-ui';
+
+export default {
+  component: AudioVisualizerWave,
+  decorators: [AgentSessionProvider],
+  render: (args: AudioVisualizerWaveProps) => {
+    const audioTrack = useMicrophone();
+
+    return <AudioVisualizerWave {...args} audioTrack={audioTrack} />;
+  },
+  args: {
+    size: 'xl',
+    lineWidth: 2,
+    state: 'connecting',
+    smoothing: 0.1,
+    color: '#1FD5F9',
+  },
+  argTypes: {
+    size: {
+      options: ['icon', 'sm', 'md', 'lg', 'xl'],
+      control: { type: 'radio' },
+    },
+    state: {
+      options: [
+        'idle',
+        'disconnected',
+        'pre-connect-buffering',
+        'connecting',
+        'initializing',
+        'listening',
+        'thinking',
+        'speaking',
+        'failed',
+      ],
```

Comment:
nitpick(non-blocking): Maybe it's worth making a constant somewhere with all these values so it's easy to update across everything if we added a new one?